src/sentry/conf/server.py

```diff
@@ -762,6 +762,13 @@ def create_partitioned_queues(name):
         ),
         "options": {"expires": 60 * 60 * 3},
     },
+    "schedule-verify-weekly-organization-reports": {
+        "task": "sentry.tasks.reports.verify_prepare_reports",
+        "schedule": crontab(
+            minute=0, hour=12, day_of_week="tuesday"  # 05:00 PDT, 09:00 EDT, 12:00 UTC
```

Comment:
How long do reports typically take to run?

Reply:
The scheduler takes 10-20 minutes, which is the piece we're verifying here. The actual reports take a few hours, but happen async.

Reply:
👍